linchpin/shell/__init__.py

```diff
@@ -267,8 +274,9 @@ def up(ctx, targets, run_id, tx_id, inventory_format):
 @click.option('-t', '--tx-id', metavar='tx_id', type=int,
               help='Destroy resources using the transaction ID (tx-id)',
               cls=MutuallyExclusiveOption, mutually_exclusive=["run_id"])
+@click.option('--ask-vault-pass', is_flag=True)
 @pass_context
-def destroy(ctx, targets, run_id, tx_id):
+def destroy(ctx, targets, run_id, tx_id, ask_vault_pass):
```

Comment:
See the reference above for `do_action`. `vault_pass` should not need to be passed, use the context object instead.